src/components/KeywordsResultsCard/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './styles.css';
+import { Tooltip } from '@material-ui/core'
+
+// The keyword component takes a list as an argument
+// anchor tag was used in case we decide that clicking
+// the keyword would produce a search specific to that word
+
+// Currently I am getting phrases in the keyword container
+// This breaks the UI if the string is too long, so I truncate
+// and provide a tooltip on hover
+
+// It is currently truncating the keyword list due to longer lists
+// not fitting in the card.
+// Next step would be to add a slider that allows us to view all keys
+
+export const KeywordsDisplay = ({ keywords }) => {
+    return (
+        <span className="keywords-container">
+            <span className="keywords">{keywords.slice(0, 5).map((keyword, index) => {
```

Comment:
@vix993 What is the intention behind this slice?

Reply:
So right now, with the data I have, the list of keywords sometimes can be long enough that it wont fit in the results card. My idea would be to create a slider effect that would allow us to display all the keywords and eliminate this slice. I didn't have time yet to make that yet but I think I can use react-slick to achieve it -> https://react-slick.neostack.com/